### ISSUE TITLE
default to property name for title

### ISSUE THREAD

**Issue by red8888** [adobe/jsonschema2md#371]
I don't want to have to add a title for every object so I don't see "Untitled object" everywhere. Its really redundant because all my object names are what I would title them anyway. Is there config option to just plug in the object name in the docs if title field is absent?

**Comment by trieloff** [comment]
No, but this would be a good default if you are in the mood for a PR.

**Comment by chrisdeely** [comment]
I have been attempting to implement this enhancement via [modifications](https://github.com/chrisdeely/jsonschema2md/commit/cd1ce34e5356cb3fe4c59a1ab544d3b7d0f9ea2f) to the `schemaProxy`, but it is clunky an error prone. 

I'm not familiar enough with the patterns in use, but it seems that a more effective path forward would be to default each element's `title` property to the element's name during the schema transversal. The approach I have in the `schemaProxy` will likely require ensuring the element being interrogated is actually a property and not a reserved keyword.
